### PR TITLE
prometheus-agent rule: add dashboard link (prometheus remote-write)

### PR DESCRIPTION
**Pull request by hervenicol**
Towards https://github.com/giantswarm/giantswarm/issues/25049

This is the dashboard I used the most when debugging prometheus-agent, thought it would be worth adding it to the alert.

### Checklist

- [x] Update CHANGELOG.md
- [ ] Add [Unit tests](https://github.com/giantswarm/prometheus-rules/#testing)
- [ ] Follow [Alert structure](https://github.com/giantswarm/prometheus-rules/#how-alerts-are-structured)
- [x] Consider [creating a dashboard](https://docs.giantswarm.io/getting-started/observability/grafana/custom-dashboards/) ([guidelines](https://intranet.giantswarm.io/docs/product/ux/guidelines/dashboards/)) (if it does not exist already) to help oncallers monitor the status of the issue.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).

### CODE REVIEW

File: helm/prometheus-rules/templates/alerting-rules/prometheus-agent.rules.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ spec:
         description: '{{`Prometheus agent remote write is failing.`}}'
         summary: Prometheus agent fails to send samples to remote write endpoint.
         opsrecipe: prometheus-agent-remote-write-failed/
+        dashboard: BG-1wO5Vk/prometheus-remote-write
```
Comment (marieroque) [inline helm/prometheus-rules/templates/alerting-rules/prometheus-agent.rules.yml:20]: Is the URL the same between each MC ?

Comment (hervenicol) [reply helm/prometheus-rules/templates/alerting-rules/prometheus-agent.rules.yml:20]: The dashboard UID and the dashboard name are the same, yes.
Alertmanager will build the address pointing to the installation's grafana.

Comment (hervenicol) [reply helm/prometheus-rules/templates/alerting-rules/prometheus-agent.rules.yml:20]: Argh, you're right:
Most dashboards have a `uid` defined (see https://github.com/giantswarm/dashboards/blob/master/helm/dashboards/dashboards/shared/private/credentials.json#L419), but some don't.
For instance, prometheus-remote-write does not have it defined.
So, grafana generates a random one.

So, I'll have to:
- update dashboard to have a `uid`
- update `prometheus-rules` documentation with this info

**Review by QuentinBisson — APPROVED** [review]

**Comment by QuentinBisson** [comment]
So, do we need dashboard uid guidelines?

**Comment by hervenicol** [comment]
> So, do we need dashboard uid guidelines?

Good question.
Today most dashboards have one because we create them in grafana, then export them. So they keep the UID grafana gave them.
But for generated dashboards (mixins) that's probably gonna be different, we may have to add them another way.

I'm not sure that's an issue for the moment, I think we can deal with this later.

**Pull request merged**